app/cafe.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        person = visitor["name"]
+
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError(person)
+
+        if visitor.get("vaccine", {}).get(
+                "expiration_date"
+        ) < datetime.date.today():

```

Comment:
Can not compare None and datetime
.get can return None

Reply:
if visitor.get("vaccine", {}).get("expiration_date") not None and  visitor.get("vaccine", {}).get("expiration_date") < datetime.date.today():

Reply:
Please let me know if one of below slutions will be ok:

1) if visitor.get("vaccine", {}).get("expiration_date") is None or visitor.get("vaccine", {}).get("expiration_date") < datetime.date.today(): 

2) vaccine_date = visitor.get("vaccine", {}).get("expiration_date") 
    if vacine_date is None or vacine_date <datetime.date.today():